presto-docs/src/main/sphinx/connector/jmx.rst

```diff
@@ -18,12 +21,27 @@ To configure the JMX connector, create a catalog properties file
 
     connector.name=jmx
 
+To enable periodical dumps, define following properties:
+
+.. code-block:: none
+
+    jmx.dump-tables=java.lang:type=Runtime,com.facebook.presto.execution.scheduler:name=NodeScheduler
+    jmx.dump-period=10s
+    jmx.eviction-limit=86400
+
+``dump-tables`` is a comma separated list of Managed Beans (MBean) that will
+be dumped every ``dump-period`` milliseconds. Dump will be with limited size
```

Comment:
I have rephrased it, if it's still unclear, can you propose something better? I'm not a native speaker neither I use English as my daily language :(